charts/jiralert/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: jiralert
 description: A Helm chart for Kubernetes to install jiralert
 type: application
-version: 1.3.0
+version: 1.4.0
```

Comment:
I think bumping to patch version is better since this PR just fixes the some security syntax and capabilities. No new feature introduced.